File: beecrowd/JavaScript/1011.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var raioEsfera = parseFloat(lines.shift());
const PI = 3.14159;

var volumeEsfera = (4 / 3.0) * PI * Math.pow(raioEsfera, 3);

console.log('VOLUME = ' + volumeEsfera.toFixed(3));

File: beecrowd/C/1011.c
#include <stdio.h>
const double PI = 3.14159;
int main() {
	double raioEsfera, volumeEsfera;
	scanf("%lf", &raioEsfera);
	
	volumeEsfera =  (4.0/3)*PI*(raioEsfera*raioEsfera*raioEsfera);
	printf("VOLUME = %.3lf\n", volumeEsfera);

	return 0;	
}